cfy_manager/utils/network.py

```diff
@@ -129,3 +129,27 @@ def check_http_response(url, **request_kwargs):
         response = e
 
     return response
+
+
+def is_ipv6(addr):
```

Comment:
There might be a util function for this in the ipaddr library or something?

Reply:
This method might be implemented using try/except on constructing `IPv6Address` from https://docs.python.org/3/library/ipaddress.html.  I didn't find any easy utility function to be imported.